samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages.properties

```diff
@@ -89,6 +89,7 @@ loading = Question loading...
 # CALCULATED_QUESTION
 calculated_question=Calculated Question
```

Comment:
I'm sorry for this request, but I'd like all the new strings at the end of the file with the '#S2U-11' comment, this will make cherry-picks easier, and also branch reconstructions.

Thanks!

Reply:
Added at the end